services/order.py

```diff
@@ -0,0 +1,34 @@
+from db.models import Order, Ticket
+from django.db import transaction
+from django.contrib.auth import get_user_model
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> None:
+    user = get_user_model().objects.get(username=username)
+    order = Order.objects.create(user=user)
+
+    if date:
+        order.created_at = date
+        order.save()
+
+    for ticket in tickets:
+        Ticket.objects.create(movie_session_id=ticket["movie_session"],
+                              order=order,
+                              row=ticket["row"],
+                              seat=ticket["seat"]
+                              )
+
+    order.save()
```

Comment:
The save() method is being called twice in this function. Once in line 15 and again in line 24. According to the checklist, the save() method should be called only once if possible. You can refactor this code to ensure that the save() method is called only once. This will improve the performance of your code.